Zend/zend_inheritance.c

```diff
@@ -619,9 +619,7 @@ static void do_inheritance_check_on_method(zend_function *child, zend_function *
 		zend_string *method_prototype = zend_get_function_declaration(parent);
 		zend_string *child_prototype = zend_get_function_declaration(child);
 
-		if (child->common.prototype && (
-			child->common.prototype->common.fn_flags & ZEND_ACC_ABSTRACT
-		)) {
+		if (parent->common.fn_flags & ZEND_ACC_ABSTRACT) {
```

Comment:
I see, so basically [this test](https://github.com/pmmaga/php-src/blob/0a96ba6131167c5d4b212cbdbc168811fc3cb8f6/Zend/tests/bug62358.phpt) should still fatal. Correct?